sdk/communication/communication-network-traversal/CHANGELOG.md

```diff
@@ -1,5 +1,14 @@
 # Release History
 
+## 1.1.0-beta.1 (2022-03-10)
+
+### Features Added
+
+- Adding optional parameter to GetRelayConfiguration to choose credential Time-To-Live in seconds of max 48 hours.
+  The default value will be used if given value exceeds it.
+
+### Breaking Changes
```

Comment:
We'll have to mention that `user` and `routeType` are now part of the options as a breaking change

Reply:
Ohh that's true!